File: src/arduinobot_py_examples/nodes/simple_parameter_node.py
#!/usr/bin/env python3

import rclpy
from arduinobot_py_examples.simple_parameter import SimpleParameter


def main(args=None):
    rclpy.init(args=args)
    node = SimpleParameter()
    rclpy.spin(node=node)
    rclpy.shutdown()


if __name__ == '__main__':
    main()

File: src/arduinobot_cpp_examples/nodes/simple_parameter_node.cpp
/**
 * @file simple_parameter_node.cpp
 * @author vaibhav
 * @brief
 * @version 0.1
 * @date 2023-11-04
 *
 *
 */

#include "arduinobot_cpp_examples/simple_parameter.h"

int main(int argc, char **argv)
{
    rclcpp::init(argc, argv);
    auto node = std::make_shared<SimpleParameter>();

    rclcpp::spin(node);
    rclcpp::shutdown();

    return 0;
}